client/belga/index.tsx

```diff
@@ -4,6 +4,9 @@ import {IUser} from 'superdesk-api'
 import {startApp} from 'superdesk-core/scripts/index';
 import belgaImage from './image';
 import belga360Archive from './360archive';
+import planningExtension from '../node_modules/superdesk-planning/client/planning-extension/dist/src/extension';
+import markForUserExtension from '../node_modules/superdesk-core/scripts/extensions/markForUser/dist/src/extension';
```

Comment:
just wondering - wouldn't import from `'superdesk-core/scripts/extensions/markForUser'` work?

Reply:
yep, it does work, I'll use that :)